src/components/Header/Header.js

```diff
@@ -1,14 +1,49 @@
-import './Header.css'
+import styled from 'styled-components'
+
 
 function Header() {
   return (
-    <div className="header">
-      <section className="header__grid-container">
-      <h2 className="header__grid-container">Adventure'n'Chill</h2>
-      <p className="header__grid-container">logout</p>
-      </section>
-    </div>
+    <StyledHeader>
+      <HeaderElements>
```

Comment:
This could be reduced to one element
```suggestion
    <StyledHeader>
```